pubspec.yaml

```diff
@@ -7,14 +7,11 @@ environment:
 
 dependencies:
   collection: ^1.14.11
-  # Dart 2.7 needs 0.39.x to resolve,
-  # and Dart 2.12+ needs 0.42.x to resolve to build_web_compilers 2.12.0, etc.
-  # to enable proper opting out of null-safety.
-  analyzer: '>=0.39.0 <0.42.0'
+  analyzer: '>=0.40.0 <2.0.0'
   build: '>=1.0.0 <3.0.0'
   built_redux: ^7.4.2
```

Comment:
`built_redux < 8.0.0` does not allow analyzer 1.0. Should this be expanded?

```suggestion
  built_redux: ">=7.4.2 <9.0.0"
```

Reply:
Based on [this CI run](https://github.com/Workiva/over_react/runs/6374857931?check_suite_focus=true) it seems like it does:
<img width="579" alt="Screen Shot 2022-05-10 at 10 52 37 AM" src="https://user-images.githubusercontent.com/1713967/167691753-b733c80d-85ee-4f39-812f-f6bdedaf98b0.png">


Though, I'm not opposed to raising the bound on built_redux so this package isn't a bottleneck. Let me take a look

Reply:
yeah we've backpatched built_redux v7 to allow analyzer v1